app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center,
+            clean_power,
+            average_rating,
+            count_of_ratings
+    ):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars):
+        income = 0
+        for car in cars:
+            if self.clean_power > car.clean_mark:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+
+        return income
+
+    def calculate_washing_price(self, car):
+        washing_volume = self.clean_power - car.clean_mark
+        rating_with_dist = self.average_rating / self.distance_from_city_center
+        washing_price = car.comfort_class * washing_volume * rating_with_dist
+
+        return round(washing_price, 1)
+
+    def wash_single_car(self, car):
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
```

Comment:
Reverse your 38th line.
In 39 line you have 'car.clean_mark = self.clean_power.
So, right 'car.clean_mark' on right for comfortable readability

Reply:
if car.clean_mark < self.clean_power:
    car.clean_mark = self.clean_power